daemons/controld/controld_execd.c

```diff
@@ -1648,6 +1648,18 @@ construct_op(const lrm_state_t *lrm_state, const xmlNode *rsc_op,
         op->interval_ms = 0;
     }
 
+    /* If this is a start action with the special expire_attrs meta-attribute
+     * set to false, go ahead and clear the purge bit from from the connection
+     * resource's flags.
+     */
+    if (pcmk__str_eq(crm_meta_value(params, PCMK_META_ATTR_EXPIRE), "false", pcmk__str_none)) {
```

Comment:
Use crm_is_true() -- even though we control the value, it's better to accept the full allowed boolean strings

Reply:
Fixed.